postgres/init/20-portal.sql

```diff
@@ -360,6 +371,16 @@ CREATE TABLE portal.company_user_assigned_app_favourites (
 );
 
 
+--
+-- Name: company_user_assigned_business_partners; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.company_user_assigned_business_partners (
+    business_partner_number character varying(20) NOT NULL,
+    company_user_id uuid NOT NULL
+);

```

Comment:
Tabellenbenennung ist auf jeden Fall besser als bei mir im Beispiel. Ich stolpere noch etwas über "business_partners" im table header, aber mir fällt auch nix besseres ein. Gefühlt gibt es etwas die Möglichkeit, dass man denken könnte, das andere Business Partners mir zugeordnet werden und nicht nur meine Child Companies die ja keine eigenständigen BPs sind.

Kleine "nice to have" Änderung: können wir copmany_user_id als erstes attribute nehmen und business_partner_number als 2tes. Für mich ist die company_user_id das führende Attribute.

Reply:
die Reihenfolge der Attribute kann ich nicht unmittelbar beeinflussen. Das generiert EFCore so, auch wenn ich im code die Reihenfolge der Declarationen umstelle. Da beide Spalten gemeinsamen den Primärschlüssel bilden spielt die Reihenfolge technisch auch keine Rolle.

Reply:
so, hab es doch noch gefunden woher sich die Reihenfolge der Spalten her ableitet :-)